### DISCUSSION TITLE
I don't understand how to put in sorceries.

### DISCUSSION THREAD

**Discussion by avorial** [3darkman/foundryvtt-chroniclesystem#56] [Q&A]
Can you please give an example of a default one so I can start making my own. Thank you.

**Comment by 3darkman** [comment]
some screenshots about the sorcery:

![image](https://github.com/3darkman/foundryvtt-chroniclesystem/assets/20743586/3171b35b-fdf4-4f51-abcb-45596e906870)

![image](https://github.com/3darkman/foundryvtt-chroniclesystem/assets/20743586/273cf0ca-c3ec-469f-875a-d063a03609bd)
